internal/daemon/plugin/bindata/vsp-ds/99.vendor-plugin-daemonset.yaml

```diff
@@ -25,19 +25,28 @@ spec:
         command: {{.Command}}
         args: {{.Args}}
         volumeMounts:
-        - name: vendor-plugin-sock
-          mountPath: /var/run/dpu-daemon/
-        - name: host-opt
-          mountPath: /opt/p4
+        - mountPath: /var/run/
+          name: vendor-plugin-sock
+        - mountPath: /opt/p4/p4-cp-nws/var
```

Comment:
Shouldn't /opt/p4 be good?

Reply:
Yeah I think so,  just opted for the more specific path, I could update this to /opt/p4